lib/icedfrisby.js

```diff
@@ -626,36 +626,66 @@ class Frisby {
    * @return {object}
    * @desc HTTP header expect helper
    */
-  expectHeader (header, content) {
+  expectHeader (header, content, options = {}) {
     const self = this
 
     header = (header + "").toLowerCase()
+    options = {
+      allowMultipleHeaders: options.allowMultipleHeaders || false
+    }
 
     this.current.expects.push(function () {
+      if(!options.allowMultipleHeaders && self.current.response.headers[header] instanceof Array){
+        throw new Error("Header '" + header + "' present more than once in HTTP response. Pass {allowMultipleHeaders: true} in options if this is expected.")
+      }
       if (typeof self.current.response.headers[header] !== "undefined") {
-        chai.expect(self.current.response.headers[header].toLowerCase()).to
-          .equal(content.toLowerCase())
+        var responseHeaders = [].concat(self.current.response.headers[header])
+        if(content instanceof RegExp){
+          chai.expect(responseHeaders).to
+            .include.something.that.matches(content)
+        }
+        else if(typeof content === "string"){
+          responseHeaders = responseHeaders.map(thisHeader => thisHeader.toLowerCase())
+          chai.expect(responseHeaders).to
+            .include.a.thing.that.equals(content.toLowerCase())
+        }
+        else {
+          throw new Error("Content '" + content + "' is neither a string or regex")
+        }
       } else {
         throw new Error("Header '" + header + "' not present in HTTP response")
       }
     })
     return this
   }
 
-
   /**
    * @param {string} header - header key
    * @param {string} content - header value content
    * @return {object}
    * @desc HTTP header expect helper (using 'contains' instead of 'equals')
    */
-  expectHeaderContains (header, content) {
+  expectHeaderContains (header, content, options) {
     const self = this
+
     header = (header + "").toLowerCase()
+    options = {
+      allowMultipleHeaders: ((options && options.hasOwnProperty('allowMultipleHeaders')) ? options.allowMultipleHeaders : false)
```

Comment:
... so we can simplify this line

Reply:
Facepalm. Of course it needs to be there too. Fixed.